src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -140,4 +146,41 @@ public void getProjectName_ReturnsProjectName() {
         // then
         assertThat(name).isEqualTo(projectName);
     }
-}
\ No newline at end of file
+
+    @Test
+    public void getTagsToExcludeFromChart_ReturnsEmptyList() {
+        // give
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+
+        // when
+        Collection<Pattern> patterns = configuration.getTagsToExcludeFromChart();
+
+        // then
+        assertThat(patterns).isEmpty();
+    }
+
+    @Test
+    public void getTagsToExcludeFromChart_addPatterns_ReturnsListWithAllPatterns() {
```

Comment:
Since the commit is already merged (thanks :dancer: )
do you still want me to make this change?